Source/DfBAdminToolkit/Presenter/ProvisioningPresenter.cs

```diff
@@ -255,7 +255,9 @@ private string ProvisionRoles(IProvisioningModel model, IMainPresenter presenter
                     }
                     else
                     {
-                        errorMessage = ErrorMessages.FAILED_TO_ADD_MEMBER;
+                        item.ProvisionStatus = response.Message;
+                        presenter.UpdateProgressInfo(response.Message);
```

Comment:
does this mean the error only shows briefly in the progress info area?  as in, if the next user provisions correctly, does the error message disappear?